files/zh-cn/web/javascript/reference/global_objects/array/with/index.md

```diff
@@ -10,7 +10,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Array/with
 ## 语法
 
 ```js-nolint
-array.with(index, value)
+with(index, value)
```

Comment:
Let's follow the notation in en-US content.

https://github.com/mdn/content/blob/7b2764468c8168bb0b3bff7823b8295ac68ad7be/files/en-us/web/javascript/reference/global_objects/array/with/index.md?plain=1#L15

```suggestion
arrayInstance.with(index, value)
```

Reply:
> Let's follow the notation in en-US content.
> 
> https://github.com/mdn/content/blob/7b2764468c8168bb0b3bff7823b8295ac68ad7be/files/en-us/web/javascript/reference/global_objects/array/with/index.md?plain=1#L15

this is inconsistent with the writing of other functions in Array.
for example, sort, some, etc. are all written directly.
<img width="759" alt="image" src="https://github.com/mdn/translated-content/assets/29879262/d637eca6-f3b9-4eaa-86d6-393543dbfeaa">
<img width="589" alt="image" src="https://github.com/mdn/translated-content/assets/29879262/7a6b1845-b0ee-4215-aad9-0afad00d15a3">

Reply:
We use `methodName(args...)` as the new style of syntax block. @fuchunhui could you align the style of en-US docs first. And then we are able to merge this (as the translated-content is the downstream of mdn/content)

Reply:
ok, right now.

Reply:
https://github.com/mdn/content/pull/32575

Reply:
I have had the same confusion in the past year. I think it may be because `with` is a keyword if I don't have a wrong memory. Maybe a comment in the docs is necessary.

Reply:
`with` do crash with the already [existed keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with), though deprecated. Maybe the notation in current en-US content can be a workaround.

Reply:
Josh may be busy with courses recently, let's wait a few days. It seems to be a good idea to add corresponding comments in context for the en-US documentation.